frontend/src/features/projects/erp/forms/AgencyResponseForm.tsx

```diff
@@ -69,16 +70,25 @@ const AgencyResponseForm = ({ isReadOnly }: IAgencyResponseFormProps) => {
               field="ninetyDayNotificationSentOn"
             />
           </Form.Row>
+          <Form.Row>
+            <Form.Label>
+              <strong>Note: </strong>Changing the dates on these fields does not change the date the
+              notifications are sent from the Common Hosted Email Service (CHES), which is managed
+              by the Exchange Lab.
+            </Form.Label>
+            <Form.Label>
+              The only way to stop future unsent notifications is to cancel the entire project.
```

Comment:
We probably should highlight that you can also cancel future notifications for specific ministries by adding a response and choosing "Not Interested".

Reply:
good idea, I'll add that